.gitignore

```diff
@@ -1,35 +1,16 @@
-vendor
-
 # Binaries for programs and plugins
-*.exe
-*.exe~
-*.dll
-*.so
-*.dylib
-bin/*
-testbin/*
-Dockerfile.cross
-
-# Test binary, build with `go test -c`
-*.test
+bin/
 
-# Output of the go coverage tool, specifically when used with LiteIDE
+# Output of the go coverage tool
 *.out
 
 # Release output
-dist/**
 operator-controller.yaml
-install.sh
 
 # Kubernetes Generated files - skip generated files, except for vendored files
 
 !vendor/**/zz_generated.*
 
-# editor and IDE paraphernalia
-.idea
```

Comment:
I use GoLand, which creates this directory in the repo root. Please leave this in.